tensorflow/examples/image_retraining/retrain.py

```diff
@@ -73,6 +73,8 @@
 import sys
 import tarfile
 
+from PIL import Image
```

Comment:
Might want to add in the comments above that PIL would need to be installed in order to be able to run this.